tests/ztimer_periodic/Makefile

```diff
@@ -10,4 +10,13 @@ TEST_ON_CI_BLACKLIST += native
 USEMODULE += fmt
 USEMODULE += ztimer_usec ztimer_msec
 
+# use periph_rtt backend for ZTIMER_MSEC for some boards to increase test coverage.
+# note: Only add board if RTT_FREQUENCY >= 1 kHz
+BOARDS_USE_RTT += nucleo-f767zi
+BOARDS_USE_RTT += samr21-xpro
```

Comment:
All `sam0 stm32 and nrf5x` can do this, how bout modifying the test so that:

```
ifneq (,$(filter stm32 nrf52 sam%,$(CPU)))
  RTT_FREQUENCY ?= RTT_MAX_FREQUENCY
  CFLAGS += -DRTT_FREQUENCY=$(RTT_FREQUENCY)
endif
```
This should force around 32Khz, which will make the test run. Then add for the test `Makefile.board.dep`
```
ifneq (,$(filter stm32 nrf52 sam%,$(CPU)))
  FEATURES_OPTIONAL += periph_rtt
  ifneq (,$(filter periph_rtt,$(FEATURES_USED)))
    USEMODULE += ztimer_periph_rtt
  endif
endif
```

Reply:
Let's give it a try :-)